docs/components/modeler/web-modeler/additional-features/call-activity-linking.md

```diff
@@ -0,0 +1,27 @@
+---
+id: call-activity-linking
+title: Call activity linking
+description: Call activity linking
+---
+
+You can use one of the following approaches to link a [call activity](/components/modeler/bpmn/call-activities/call-activities.md) to another process.
+
+### Using the link button
+
+If you select a call activity, a link button appears at the bottom right. Clicking on the button opens a dialog from which you can choose any diagram from the same project. Click on the `Link` button after selecting the diagram. This will create the link by copying the process ID of the diagram to the `Called element` section in the properties panel.
+
+![overlay](img/overlay.png)
+
+For call activities that are already linked, the button appears even when the element is not selected. Clicking on the button opens a dialog which shows the diagram that the call activity is linked to. It is possible to navigate to the linked diagram by clicking on it or you can use the `Unlink` button to remove the link.
```

Comment:
This does not cover the (edge) case of multiple files with same process ids found. Do we want to cover edge cases in the docs in general?
Also, it should be "...which shows _the name of the diagram_ that the call activity _links to_."

Reply:
I tried to keep the doc simple and included the bare minimum necessary for a user to understand how to use this feature. For the edge cases, we have included contextual help in the application in the form of `info` messages that appear when the user encounters them.